### PR TITLE
Create a gatk-variantcalling pipeline.

### PR DESCRIPTION
**Pull request by rhpvorderman**

### CODE REVIEW

File: gatk-variantcalling.wdl [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+            referenceFastaDict = referenceFastaDict,
+            scatterSize = scatterSize,
+            regions = regions,
+            dockerImage = dockerImages["biopet-scatterregions"]
+    }
+
+    # Glob messes with order of scatters (10 comes before 1), which causes problems at gatherGvcfs
+    call biopet.ReorderGlobbedScatters as orderedScatters {
+        input:
+            scatters = scatterList.scatters,
```
Comment (DavyCats) [inline gatk-variantcalling.wdl]:
```suggestion
            scatters = scatterList.scatters
```

File: gatk-variantcalling.wdl [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+            input:
+                bam=bam.file,
+                bamIndex=bam.index,
+                scatterList=orderedScatters.reorderedScatters,
+                referenceFasta=referenceFasta,
+                referenceFastaDict=referenceFastaDict,
+                referenceFastaFai=referenceFastaFai,
+                dbsnpVCF=dbsnpVCF,
+                dbsnpVCFIndex=dbsnpVCFIndex,
+                outputDir=outputDir + "/samples/" + basename(bam.file, ".bam")
```
Comment (DavyCats) [inline gatk-variantcalling.wdl]: [Please add spaces around the `=`](https://biowdl.github.io/styleGuidelines.html#expressions)

File: gatk-variantcalling.wdl [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+                bam=bam.file,
+                bamIndex=bam.index,
+                scatterList=orderedScatters.reorderedScatters,
+                referenceFasta=referenceFasta,
+                referenceFastaDict=referenceFastaDict,
+                referenceFastaFai=referenceFastaFai,
+                dbsnpVCF=dbsnpVCF,
+                dbsnpVCFIndex=dbsnpVCFIndex,
+                outputDir=outputDir + "/samples/" + basename(bam.file, ".bam"),
+                dockerImages=dockerImages
```
Comment (DavyCats) [inline gatk-variantcalling.wdl]: [Please add spaces around the `=`.](https://biowdl.github.io/styleGuidelines.html#expressions)

File: gatk-variantcalling.wdl [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+            dockerImage = dockerImages["picard"]
+    }
+
+    output {
+        File outputVcf = gatherVcfs.outputVcf
+        File outputVcfIndex = gatherVcfs.outputVcfIndex
+        File? outputGVcf = gatherGvcfs.outputVcf
+        File? outputGVcfIndex = gatherGvcfs.outputVcfIndex
+    }
+}
```
Comment (DavyCats) [inline gatk-variantcalling.wdl]: Please add a parameter_meta section.

File: Jenkinsfile [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+            slackSend(color: '#FF0000', message: "Failure: Job '${env.JOB_NAME} #${env.BUILD_NUMBER}' (<${env.BUILD_URL}|Open>)", channel: '#biopet-bot', teamDomain: 'lumc', tokenCredentialId: 'lumc')
+        }
+        unstable {
+            slackSend(color: '#FFCC00', message: "Unstable: Job '${env.JOB_NAME} #${env.BUILD_NUMBER}' (<${env.BUILD_URL}|Open>)", channel: '#biopet-bot', teamDomain: 'lumc', tokenCredentialId: 'lumc')
+        }
+        aborted {
+            slackSend(color: '#7f7f7f', message: "Aborted: Job '${env.JOB_NAME} #${env.BUILD_NUMBER}' (<${env.BUILD_URL}|Open>)", channel: '#biopet-bot', teamDomain: 'lumc', tokenCredentialId: 'lumc')
+        }
+        success {
+            slackSend(color: '#00FF00', message: "Success: Job '${env.JOB_NAME} #${env.BUILD_NUMBER}' (<${env.BUILD_URL}|Open>)", channel: '#biopet-bot', teamDomain: 'lumc', tokenCredentialId: 'lumc')
```
Comment (DavyCats) [inline Jenkinsfile]: Wasn't this slack stuff removed at some point? Is this an outdated Jenkinsfile?

**Review by DavyCats — CHANGES_REQUESTED** [review]

File: .travis.yml [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+script:
+  - pytest --tag $TEST tests/
+
+matrix:
+  include:
+    - env: TEST=single_sample
+    - env: TEST=two_sample
+    # Below is for testing if all files are valid and if all submodules are up to date.
+    - env: TEST=Womtool validate and submodule up to date
```
Comment (Redmar-van-den-Berg) [inline .travis.yml:33]: Don't you need to escape the spaces in `TEST` for `TEST=Womtool validate and submodule up to date`

Comment (rhpvorderman) [reply .travis.yml:33]: This environment variable is not used except as a label for travis. Technically it is not needed.

**Review by DavyCats — APPROVED** [review]

**Pull request merged**